client/my-sites/media-library/list-item-image.jsx

```diff
@@ -80,6 +80,7 @@ module.exports = React.createClass( {
 		var url = MediaUtils.url( this.props.media, {
 			photon: this.props.thumbnailType === MEDIA_IMAGE_PHOTON,
 			maxWidth: this.props.maxImageWidth,
+			size: this.props.thumbnailType === MEDIA_IMAGE_THUMBNAIL ? 'medium' : false,
```

Comment:
I like your proposal of making explicit that we can either use photon, resize, or reusing existing thumbnails through these series of PRs. I guess that your next step will be introducing the logic for the thumbnail type in the [content component](https://github.com/Automattic/wp-calypso/blob/add/thumbnail-media-library-type/client/my-sites/media-library/content.jsx#L167), so I was wondering whether it'd make sense to improve the MediaUtils library and modify its API to use these same concepts instead passing it finer-grained params that contain logic (the fact that we only pass the size if we want to reuse existing thumbnails was a bit obscure to me).